stylesheets/blue/components/_nav.scss

```diff
@@ -249,5 +258,10 @@ $Nav-item-color-hover: $Theme-color-yellowOrange;
   .Nav-menu {
     display: flex;
     align-items: center;
+
+    padding-right: $Theme-spacing-small;
+    padding-left: $Theme-spacing-small;
+    margin-right: -$Theme-spacing-small;
+    margin-left: -$Theme-spacing-small;
```

Comment:
I'm not a big fan of negative margins, isn't there any other way? what do you think of putting the link outside?